docs/dns-policy.md

```diff
@@ -94,8 +94,8 @@ spec:
       - 201
       - 301
     failureThreshold: 5
-    port: 443
-    protocol: https
+    port: 80
+    protocol: http
```

Comment:
> Given this is an example DNSPolicy, is the reason for the change to http to avoid having to reference TLSPolicy as well?

Yes, since this is DNSPolicy doc i didn't want to talk about TLSPolicy in it and you'd need to add a TLSPolicy now to make it work for https. Also i don't see it needs to, DNS doesn't care if your endpoints are secure or not.